docs-checks fails due to OutOfMemoryError

```
Terminating due to java.lang.OutOfMemoryError: Java heap space
.github/bin/retry: retrying [./mvnw install -B --strict-checksums -V -T C1 -DskipTests -P ci -am -pl :trino-docs] timed out
Error: Process completed with exit code 1.
```
https://github.com/trinodb/trino/runs/4709585911

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -55,6 +55,7 @@ jobs:
           echo "::add-matcher::.github/problem-matcher.json"
       - name: Maven Checks
         run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN install -B --strict-checksums -V -T C1 -DskipTests -P ci -am -pl ':trino-docs'
 
   test-docs:

